packages/flex-plugin-scripts/src/clients/builds.ts

```diff
@@ -1,9 +1,32 @@
-import { Credential, logger } from '@twilio/flex-dev-utils';
+/* eslint-disable camelcase */
+
+import { stringify } from 'querystring';
+
+import { logger, TwilioCliError, urlJoin } from '@twilio/flex-dev-utils';
 import { isSidOfType, SidPrefix } from '@twilio/flex-dev-utils/dist/sids';
 
-import BaseClient from './baseClient';
-import { Build, BuildStatus } from './serverless-types';
-import ServiceClient from './services';
+import ServerlessClient from './serverless-client';
+import { AssetVersion, FunctionVersion } from './assets';
+
+export enum BuildStatus {
+  Building = 'building',
+  Completed = 'completed',
+  Failed = 'failed',
+}
+
+export interface ServerlessBuild {
+  sid: string;
+  account_sid: string;
+  url: string;
+  date_updated: string;
+  date_created: string;
+  status: BuildStatus;
+  asset_versions: AssetVersion[];
+  function_versions: FunctionVersion[];
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
according to the rule, `object` should be an accepted type. or you could do `Record<string, string>` (or whatever the key/value types should be)